docs/openapi/oad/castor.oad.yaml

```diff
@@ -0,0 +1,390 @@
+openapi: 3.1.0
+info:
+  title: Castor Service API
+  description: |
+    This API allows to interact with the Castor service. It provides methods to reserve and consume tuples, and to get
+    telemetry data. The service endpoints are separated into two categories: inter-VCP and intra-VCP.
```

Comment:
"inter-VCP and intra-VCP" is not needed here. It's part of the following enumeration already.